pkg/cmd/admin/router/router.go

```diff
@@ -158,7 +166,17 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		}
 	}
 
-	ports, err := app.ContainerPortsFromString(cfg.Ports)
+	// For host networking, use the default ports 80/443. We don't have
+	// config options as yet to specify what the externally exposed
+	// ports are.
+	ports, err := app.ContainerPortsFromString(defaultPorts)
```

Comment:
I guess we could do that instead - it just leaves a possibility for misconfiguration in the host networking case (since you can specify a port pair there which makes sense for the container networking case). Don't really have a strong opinion either way, will change it back to the way it was.